githook/prepare-commit-msg

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+import sys
+
+def is_integer(value: str) -> bool:
+    try:
+        int(value)
+        return True
+    except ValueError as _:
+        pass
+
+    return False
+
+def check_commit_message() -> int:
+    commit_message_filename: str = sys.argv[1]
+    with open(commit_message_filename) as file:
+        first_line: str = file.readline().strip()
+
+    issue_name = first_line.split(" ")[0]
+
+    # Allow fixup commits
+    if issue_name == "fixup!":
+        return 0
+
+    if issue_name == "WIP":
+        return 0
+
+    if issue_name.startswith("KTOR-") and is_integer(issue_name[5:]):
```

Comment:
Isn't it easier to just match with a regular expression?

Reply:
Nope. I would avoid using regex in any case. Right now it's easy to read

Reply:
Cause regex not so good from a maintenance and readability perspective.
Could you tell me why 2 explicit checks should be merged in a single(which you also have to parse during reading)?